driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/DesktopWebDriverRequest.java

```diff
@@ -89,17 +123,10 @@ public Dimension getWindowSize() {
                 int height = Integer.parseInt(matcher.group(2));
                 dimension = new Dimension(width, height);
             } else {
-                dimension = getDefaultDimension();
-                log().error(String.format("Unable to parse property %s=%s, falling back to default: %s", Testerra.Properties.WINDOW_SIZE, windowSizeProperty, dimension));
+                log().error(String.format("Unable to parse property %s=%s, falling back to default: %s", Properties.WINDOW_SIZE, windowSizeProperty, dimension));
```

Comment:
~~O stranger, tell the Lacedaemonians that~~
we lie here, ~~obedient to their words.~~

The actual fallback is hard-wired into this very method.
(Note that this was an issue before as well - but I had missed it previously.)

... Mayhap `testT08_invalidWindowSize` is currently a bit too optimistic in its assumptions - it did not protect us.

Reply:
As discussed this property is too unique at the moment, that we should move the parsing to another component.

Reply:
> ~O stranger, tell the Lacedaemonians that~ we lie here, ~obedient to their words.~
> 
> The actual default is never read. (Note that this was an issue before as well - but I had missed it previously.)
> 
> ... Mayhap `testT08_invalidWindowSize` is currently a bit too optimistic in its assumptions - it did not protect us.

Now the tests useing the default value of the property.